### PR TITLE
Update RO import

### PR DESCRIPTION
**Pull request by anitacaron**
The previous RO import didn't include the new annotation property `defined by construct` and the annotation properties `never_in_taxon`, `present_in_taxon` and `mutually spatially disjoint with` needed to use `robot expand`.

Related to https://github.com/obophenotype/uberon/issues/2382

### CODE REVIEW

File: src/ontology/uberon.Makefile [hunk 1]
```diff
@@ -406,7 +406,7 @@ tmp/ro_seed.txt: imports/ro_terms.txt reports/uberon-edit-object-properties.csv
 	cat $^ | sort | uniq >  $@
 
 imports/ro_import.owl: mirror/ro.owl $(TMPDIR)/seed.owl tmp/ro_seed.txt
-	if [ $(IMP) = true ]; then $(ROBOT) extract -i $< -m BOT -T tmp/ro_seed.txt --individuals exclude annotate -O $(ONTBASE)/$@ -a $(DC)/title "Relations Ontology Module for Uberon" -o $@.tmp.owl && $(OWLTOOLS_NO_CAT) $@.tmp.owl --remove-tbox --remove-annotation-assertions -l -d -r  -o $@; fi
+	if [ $(IMP) = true ]; then $(ROBOT) extract -i $< -m BOT -T tmp/ro_seed.txt --individuals exclude annotate -O $(ONTBASE)/$@ -a $(DC)/title "Relations Ontology Module for Uberon" -o $@.tmp.owl && $(OWLTOOLS_NO_CAT) $@.tmp.owl --remove-tbox --remove-annotation-assertions -l -d -r -p http://purl.obolibrary.org/obo/OMO_0002000 -o $@; fi
```
Comment (matentzn) [inline src/ontology/uberon.Makefile:409]: How does this change make sure that all never_in_taxon and OMO:2000 are added into RO? What is -p?

**Review by matentzn — APPROVED** [review]

File: src/ontology/uberon.Makefile (hunk 1 above)
Comment (anitacaron) [reply src/ontology/uberon.Makefile:409]: This only makes sure that OMO:000200 is not removed.

Following the owltools' help (`sh run.sh owltools --remove-annotations-assertions -p -h`)

`-p IRI     if specified, all properties with IRI are preserved. Can be specified multiple times`

Comment (anitacaron) [reply src/ontology/uberon.Makefile:409]: I included the id of the annotations in the `ro_terms.txt`

Comment (matentzn) [reply src/ontology/uberon.Makefile:409]: Thank you! feel free to merge any time

**Pull request merged**